tests/test_maxcut.py

```diff
@@ -54,7 +54,26 @@ def test_maxcut_qaoa_obj_fixed_angles():
             gamma, beta, AR = get_fixed_gamma_beta(d, p, return_AR=True)
             for simulator in ["auto", "qiskit"]:
                 f = get_qaoa_maxcut_objective(N, p, G=G, parameterization="gamma beta", simulator=simulator)
-                assert -f(gamma, beta) / optimal_cut > AR
+                assert f(gamma, beta) / optimal_cut < AR
+
+
+def test_maxcut_qaoa_obj_fixed_angles_with_terms_and_precomputed_energies():
```

Comment:
This test case is to compare the result are same when either terms or precomputed energies are passed. This can be tested only with 'auto' simulator, qiskit simulator only allows terms to be passed for testing